week2-section2/exercise2/PlayList.cpp

```diff
@@ -0,0 +1,61 @@
+#include <iostream>
+#include "Song.h"
+#include"PlayList.h"
+
+	FPlaylist::FPlaylist()
+	{
+		PlaylistTitle = "";
+		songCount = 0;
+	}
+	FPlaylist::FPlaylist ( const std::string InPlayListTitle, const int InSongCount )
+	{
+		PlaylistTitle = InPlayListTitle;
+		songCount = InSongCount;
```

Comment:
The goal of SongCount is to keep track of the number of songs currently in the playlist, not to specify some max value of songs (which is what MaxSongCount is for).

That means that this constructor should not take in a parameter to initialize this value, since it should always start as 0 and increase with each added song.